d3rlpy/metrics/scorer.py

```diff
@@ -169,7 +169,7 @@ def discounted_sum_of_advantage_scorer(
 
             total_sums += sum_advantages
 
-    # smaller is better
```

Comment:
This comment was for explaining why the negative sign is here. So if you update this line, could you include that?

Reply:
Hi, thanks for your reply. We can't understand why `d3rlpy` returns the negative scale of metrics in `NEGATIVE_SCORERS` and then multiply `-1` when evaluating and recording(https://github.com/takuseno/d3rlpy/blob/f3b3f55980a3ca557a7e503b9acbfa42b6974c0a/d3rlpy/base.py#L796).

In short, I have the following questions:

* Why can't return the raw values directly rather than the negative scale ones. This is not friendly for users to understand and add custom scorers.

* Could you please explain the reason why designing the `NEGATIVE_SCORERS`.

Reply:
Hi. The reason is that the scorer functions are compatible scikit-learn scorer functions where the larger positive values mean better so that d3rlpy can be used with the tons of scikit-learn utilities. I'm planning that we'll drop this compatibility in the future. But I want to keep it for now.

Reply:
Also, users can freely add their own scorer functions unless they use scikit-learn utilities.

Reply:
I believe this comment also tells the reason.
https://github.com/takuseno/d3rlpy/blob/f3b3f55980a3ca557a7e503b9acbfa42b6974c0a/d3rlpy/base.py#L794